projects/components/src/file-upload/file-upload.component.ts

```diff
@@ -46,6 +46,13 @@ import { FileTypeUtil, SupportedFileType } from './file-types';
 
     <ng-template #defaultSubTpl>
       <ul class="file-requirements-list">
+        <li class="item" *ngFor="let req of (this.config?.customRequirementsInfo ?? [])">
+          <ht-icon class="icon" icon="${IconType.CheckCircleFill}" size="${IconSize.ExtraSmall}"></ht-icon>
+          <div class="text">
+            <strong>{{ req.key }}</strong
```

Comment:
need `htDisplayString` for `key` also?

Reply:
Not necessary I think. But not harm in adding.

Reply:
+1

Reply:
Added.